README.md

```diff
@@ -54,10 +54,30 @@ Flags:
                                 Metrics Namespace ($METRICS_NAMESPACE)
   --metrics_path="/metrics"     Path under which to expose metrics for the bridge ($METRICS_PATH)
   --extended_details            When enabled, alerts are presented in HTML format and include colorized status (FIR|RES), alert start time, and a link to the generator of the alert ($EXTENDED_DETAILS)
+  --dispatch_errors             When enabled, alerts will be tried to dispatch with a error-message regarding faulty templating or missing fields to help debugging ($DISPATCH_ERRORS)
   --debug                       Enable debug output of the server
   --version                     Show application version.
 ```
 
+### Templating
+The bridge now supports [Go templating](https://golang.org/pkg/text/template/), so you can customize the alertmessages further.  
+For example add following line to the title:  
+`{{if eq .Status "firing"}}🔥{{else}}✅{{end}}`  
+This differentiates firing from resolving alerts.  
+  
+Also, there are two methods you can use for additional customisation:
+```
+.Values                 Access alert-values, -labels and -metrics. 
```

Comment:
I'm having trouble envisioning how this would work in practice. Can you add an example to this section of the doc?

Reply:
I added an example. These methods are kind of a hacked version to get further data form grafana like actual alert values and labels of firing alerts. This helps to know for example which individual job or instance fired an alert.